packages/blockchain-wallet-v4-frontend/src/modals/Onboarding/LinkFromExchangeAccount/template.conflict.tsx

```diff
@@ -0,0 +1,347 @@
+import { Button, Icon, Text } from 'blockchain-info-components'
+import { ButtonWrapper, MainWrapper } from './styles'
+import { FormattedHTMLMessage, FormattedMessage } from 'react-intl'
+import { Props } from '.'
+
+import React, { PureComponent } from 'react'
+import styled from 'styled-components'
+
+const HeadingContainer = styled.div`
+  width: 100%;
+  margin-bottom: 36px;
+`
+const FullWidthWrapper = styled.div`
+  width: 100%;
+  display: flex;
+`
+const TitleWrapper = styled.div`
+  margin-top: 8px;
+  width: 100%;
+  display: flex;
+`
+const SectionWrapper = styled.div`
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+`
+const RadioInput = styled.input`
+  width: 24px;
+  height: 24px;
+`
+const RadioInputLabel = styled.label`
+  display: flex;
+  flex: 1;
+`
+const EmailContainer = styled.div`
+  display: flex;
+  justify-content: space-between;
+  flex: 1;
+  margin: 0 0 16px 16px;
+`
+const AddressContainer = styled(EmailContainer)`
+  flex-direction: column;
+`
+
+type State = {
+  allowSubmit: boolean
+  considerAddress: boolean
+  considerEmail: boolean
+  selectedAddress: string | null
+  selectedEmail: string | null
+}
+
+class Conflict extends PureComponent<
+  Props & { close: () => void; error: any },
```

Comment:
Are you able to add an LinkExchangeErrorType by any chance?

Reply:
Yup I moved it 👍